benchmark/Program.cs

```diff
@@ -21,6 +21,7 @@ static void Main(string[] args)
             var benchmarkConfig = ManualConfig.Union(DefaultConfig.Instance, new BenchmarkConfig());
 #endif
             BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args, benchmarkConfig);
+            BenchmarkRunner.Run<GraphServiceClientBenchmark>();

```

Comment:
This doesn't look right. BenchmarkSwitcher above should select the benchmark to run, we should run it directly like this.

Reply:
yes...was just trying somethings out. I'll move this back to draft, for when I have time to come back to it. the benchmarks were not working for the other though

Reply:
reverted some things